unfoldingWord/translationCore
# Check Box Fix For Online Dialog

#### This pull request addresses:
This PR address the online action dialog having weird behavior of re-mounting after clicking the check box and also not showing expected feedback that it actually has been checked.


#### How to test this pull request:
Open the app
and trigger an action that will show the ```Online Action Dialog```
This could be trying to import a project from online or even accessing one of the urls in the app.
You should be able to check and uncheck the box without any problems.

**Note: This problem was unrelated to React 16.**
The problem occurred from the component being re-mounted by ```react-localize-redux```
They're API highly suggests not using the ```localize``` wrapper for components already wrapped by react-redux ```connect```. 
I took the liberty to make the fix for the Alert Modal Dialog but similar issues that are out of scope of this story may happen in the future if we do not account for those.
i.e. ```src/js/pages/app.js``` lines 58-59 are wrapping two containers with ```withLocale``` those are also wrapped with ```connect```.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3762)
